apps/native-component-list/app.config.js

```diff
@@ -7,29 +7,20 @@
 //   - This can be debugged with `EXPO_DEBUG=1` and `DEBUG=expo:*`
 // 5. `EXPO_SDK_VERSION=43.0.0 expo run:android`
 export default ({ config }) => {
-  config.version = '43.0.0';
   // app.json defines the sdkVersion as UNVERSIONED, we can override it here dynamically if we need to,
   // for example with an environment variable.
   if (process.env.EXPO_SDK_VERSION) {
     config.sdkVersion = process.env.EXPO_SDK_VERSION;
   }
   config.plugins = [
-    // iOS plugins
-    // Add a plugin to modify the AppDelegate.
-    './plugins/withNotFoundModule',
-    // Add the React DevMenu back to the client.
-    './plugins/withDevMenu',
-
     // Android plugins
-
     // expo-modules-test-core requires kotlin, so additional setup must be executed.
     'expo-modules-test-core',
     [
       './plugins/withGradleProperties',
       {
         // Increase default java VM size so it can handle building all the Expo packages.
-        'org.gradle.jvmargs':
-          '-Xmx3g -XX:MaxPermSize=2048m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8',
+        'org.gradle.jvmargs': '-Xmx3g -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8',
```

Comment:
```suggestion
        'org.gradle.jvmargs=-Xmx8g -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8',
```
Let's use the same values that we have in the `bare-expo` project.